### ISSUE TITLE
Define Research Plan for testing of the Core Gh ticket

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#37377]
## User Story
As a Governance team member, I want to write a detailed and thorough research plan and conversation guide for testing the new Collaboration Cycle core gh ticket so that I can ensure I test consistently, and uncover what works well and doesn't work well wrt the new core template.

## Description
Before we can begin testing the new core gh ticket, we want to establish a Research Plan so that we can be in alignment on our testing goals and objectives. We want to be able to test the usability of the new core template compared to the existing templates and see if VFS PMs are able to easily request the various Collaboration Cycle touchpoints.

## Artifacts Needing Updates
Update the Research Plan and Conversation Guide templates created in the[ Collaboration Cycle core github ticket: testing](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2098331692/Collaboration+Cycle+Core+GitHub+ticket+Testing) space on Confluence. 

## Tasks
N/A

## Acceptance Criteria
- [x] The Research Plan template is completed, peer reviewed and approved by Governance team leadership
- [x]  The Conversation Guide template is completed, peer reviewed and approved by Governance team leadership
- [x] The team has completed "dry run" testing with a Governance team member (Allison)
- [x] Initial sample tickets have been created
- [x] Kickoff ticket has only kickoff information
- [x] User Story need has been fulfilled

---

## Governance Team Resources
[Documentation Update Guidance](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2047868956/WIP+Documentation+Update+Guidance) <br />
[Guidelines for Governance Team Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1127219219/Guidelines+for+Team+spaces) <br />
[Guidelines for Platform Products and Services Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1321140361/Guidelines+for+Platform+Products+and+Services+space) <br />
[Description of Platform Website spaces and content](https://vfs.atlassian.net/wiki/spaces/AP/pages/1241579796/Organization) <br />

---
## How to configure this issue
- [ ] **Ensure that story is attached to appropriate Iniative (if not tech debt)** (what body of work is this a part of?)
- [ ] Ensure that any additional labels are added ('tech-debt', 'collaboration-cycle')
- [ ] Ensure that story is in the correct pipeline (Backlog, Planned, In Progress, Review/QA, Closed)
- [ ] Ensure that assignees are correct so you receive all updates/comments on the ticket

**Comment by seanasewell** [comment]
@briandeconinck I finished the [conversation guide](https://vfs.atlassian.net/l/c/CpMcyvHL). I added 3 tasks:

1. Fill out product information and answer kickoff questions using markdown version
2. Fill out product information and answer kickoff questions using Yaml version
3. Fill out the Midpoint Review section of the Core Collaboration Cycle Ticket

Filling out the entire ticket might be a bit redundant and doesn't really align with the typical process of filling out one touchpoint at a time. Which is why task 3 asks them to just fill out the Midpoint Review section (product information, kickoff questions, and design intent section will already be completed given the scenario). Does this seem fair given that each touchpoint is pretty similar or do we want them to fill out the entire Core Collaboration Cycle ticket all at once? 

Also, did we want to add a task asking them to complete part of the current Collaboration Cycle process? Maybe the Midpoint Review since we are asking them to fill out that section already in the Core Collaboration Cycle ticket? Or is that overkill? As of right now, the study for each participant requires 20-30 minutes of their time. I don't want to over engineer this but also want to ensure that we are capturing feedback that will help us understand which process is better. 

@shiragoodman Would love your feedback as well!

**Comment by shiragoodman** [comment]
hey @seanasewell  - I apologize for the delayed response. I had provided some feedback on the ticket (as you're aware) and set up a meeting to discuss with you, @briandeconinck and @claytonzook on Tuesday 3/29.

I agree with the 3 tasks. I don't think they need to complete the entire ticket. However, I did want to request that we add a 4th for post-staging (or post-launch) tasks. This would be where they'd provide the launch date,  production URL, add the Service Now Ticket # submitting accessibility findings to the 508 office (this is new so we will need to update the template), and essentially close out the ticket. This is a new part of the Collaboration Cycle process so we want to make sure the task we ask them to complete is clear and we have solid follow up questions.

To add to that, we will need to update the template. I am going to set up a 2nd meeting before the first for Monday 3/29 so we can edit the template together. There's a lot of little comments I have and I think that may be the quickest and easiest way to convey my thoughts rather than commenting in the doc.

Re: your other questions:
_Also, did we want to add a task asking them to complete part of the current Collaboration Cycle process? Maybe the Midpoint Review since we are asking them to fill out that section already in the Core Collaboration Cycle ticket? Or is that overkill? As of right now, the study for each participant requires 20-30 minutes of their time. I don't want to over engineer this but also want to ensure that we are capturing feedback that will help us understand which process is better._
No, I don't think we need to ask them to do that. It could be overkill.

_Also, how do we want to setup the testing environment? And include the Zenhub board?_
I like Clayton's suggestion of just DM'ing them the links for task 1 and task 2. For task 3 and new task 4, we're going to have to create sample tickets. Preferably 1 per task per participant, so if we have 8 participants, 16 total. We can discuss these details more next week when we chat. I imagine for Task 3 and Task 4, we'll DM them the Zenhub board and ask them to find their product. The sample tickets for task 3 will have information completed up to Design Intent. The sample tickets for task 4 will have information completed up to staging review. (We can fill it in with dummy data. it doesn't have to be 100% accurate). Does this help?

**Comment by briandeconinck** [comment]
End of day update following our conversation, @shiragoodman @seanasewell:

[Test repo issue templates](https://github.com/briandeconinck/cc-test/issues/new/choose) have been updated as discussed.
- Removed three fields from the Yaml version, cleaned up a few other things I noticed.
- Removed "other front-end information" from md version, added ServiceNow ticket field to md version.
- Reduced md version to just sections relevant at kickoff, making it roughly consistent with the Yaml version.

In the [full md issue template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/briandeconinck-core-cc-ticket/products/platform/vsp-collaboration-cycle/core-collaboration-cycle-ticket.md):
- Artifacts section reorganized as discussed. I think this is better!
- Cleaned up copy/paste debris for Staging Review section.
- Reordered touchpoints as discussed.

Still to-do: Update the Sitewide Content touchpoint based on their new process. Probably not required before starting testing, just don't want to forget.

**Comment by shiragoodman** [comment]
@chaseakirby can you work with @seanasewell and @briandeconinck to close this out?

**Comment by chaseakirby** [comment]
@shiragoodman Absolutely! I can help make the updates to the Release Plan based on the Research Plan and Conversation Guide,
@seanasewell and @briandeconinck I see we are on creating the sample tickets, can I help by adding info to any of the sample tickets on the ZenHub board?

**Comment by seanasewell** [comment]
@chaseakirby I made the conversation guides for all the participants that have signed up so far. I'll just need to update the links to the sample tickets for each PM. @briandeconinck is creating those this afternoon. The help might be needed but i'll let Brian speak to that.
